src/main/webapp/resources/js/shib/idpselect_config.js

```diff
@@ -139,3 +144,8 @@ function IdPSelectUIParms() {
     this.maxHeight = 69;
     this.bestRatio = Math.log(80 / 60);
 }
+
+function urlParam(name) {
+    var results = new RegExp('[\?&]' + name + '=([^&#]*)').exec(window.location.href);
+    return results[1] || 0;
```

Comment:
@oscardssmith sure, you can include this comment for the function:
Returns the value for the specified parameter, or 0 if the parameter is not found in the url string